integration/benchmark/cmd/ci/main.go

```diff
@@ -216,35 +223,23 @@ func calculateTpsSlices(start, end time.Time, leadTime, sliceTime, commit, goVer
 	return slices
 }
 
-func prepareDataForBigQuery(slices []dataSlice, ci bool) {
-	jsonText, err := json.Marshal(slices)
+func sendDataToBigQuery(
+	ctx context.Context,
+	projectName, datasetName, tableName string,
+	slices []dataSlice,
+) error {
+	bqClient, err := bigquery.NewClient(ctx, projectName)
```

Comment:
i'm not familiar with the bigquery api.  do you need some sort of close / cleanup for the client / table?